jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskdetails/TaskDetailsPresenter.java

```diff
@@ -92,31 +84,23 @@ public void callback(Void nothing) {
         }
     }
 
-    public void refreshTask() {
-
-        taskService.call(new RemoteCallback<TaskSummary>() {
-            @Override
-            public void callback(TaskSummary details) {
-                if (details == null) {
-                    setReadOnlyTaskDetail();
-                    return;
-                }
-                if (details.getStatus().equals(TASK_STATUS_COMPLETED)) {
-                    setReadOnlyTaskDetail();
-                }
-                view.setTaskDescription(details.getDescription());
-                final Long date = UTCDateBox.date2utc(details.getExpirationTime());
-                if (date != null) {
-                    view.setDueDate(date);
-                    view.setDueDateTime(date);
-                }
-                view.setUser(details.getActualOwner());
-                view.setTaskStatus(details.getStatus());
-                view.setTaskPriority(String.valueOf(details.getPriority()));
-            }
-        }).getTask(currentServerTemplateId,
-                   currentContainerId,
-                   currentTaskId);
+    protected void setTaskDetails(String status,
+                                  String description,
+                                  String actualOwner,
+                                  Date expirationTime,
+                                  String priority) {
+        if (status.equals(TASK_STATUS_COMPLETED)) {
```

Comment:
Usually is better to use the constant as the first element to compare, that way you can avoid potential NPE.